src/rust-getting-started/string-tutorial.md

```diff
@@ -0,0 +1,238 @@
+# Working with Strings in the Windows Crate
+
+Calls involving strings can be tricky; there are several different types of strings in the Windows APIs:
```

Comment:
Try to be as concise as possible. The fact that this topic exists is sufficient to indicate that strings can be tricky. 😊
```suggestion
There are several string types in the Windows APIs including:
```